product.go

```diff
@@ -12,7 +12,7 @@ type ProductAttributes struct {
 	StatusFormatted string    `json:"status_formatted"`
 	ThumbURL        string    `json:"thumb_url"`
 	LargeThumbURL   string    `json:"large_thumb_url"`
-	Price           int       `json:"price"`
+	Price           string    `json:"price"`
```

Comment:
Let's make it as `any` I have a similar incident on https://github.com/NdoleStudio/lemonsqueezy-go/commit/8da1ed401d4607e14d60919d59c543682f809488 so it works for both string and int. I think the lemonsqueezy sometimes returns strings and other times it returns int.

Reply:
agreed